Development/src/pages/logs.js

```diff
@@ -37,178 +37,156 @@ const cardActionStyle = {
     float: 'right',
 };
 
-export class EventsList extends React.Component {
-    constructor(props) {
-        super(props);
-        this.filter = '';
-        this.filterObject = {};
-        this.state = {
-            data: [],
-        };
-        this.nextPage = this.nextPage.bind(this);
-        this.setFilter = this.setFilter.bind(this);
-        this.filterPage = this.filterPage.bind(this);
-        this.firstLoad = this.firstLoad.bind(this);
-        this.firstLoad();
-    }
-
-    async firstLoad() {
+export const EventsList = () => {
+    const firstLoad = async () => {
         const params = {
             filter: {},
             pagination: { page: 1, perPage: 10 },
-            sort: { field: 'timestamp', order: 'DESC' },
+            sort: { field: 'id', order: 'DESC' },
```

Comment:
I don't think `pagination` or `sort` should be here at all, since they're not relevant for cursor-based pagination with our `dataProvider`?

Reply:
These params are actually present on all list views, as it has no effect on our dataProvider, I will remove the extraneous lines.